Detectors/ITSMFT/ITS/tracking/src/TrackerTraits.cxx

```diff
@@ -821,9 +821,9 @@ bool TrackerTraits::fitTrack(TrackITSExt& track, int start, int end, int step, f
       return false;
     }
 
-    if (!propInstance->propagateToX(track, trackingHit.xTrackingFrame, getBz(), o2::base::PropagatorImpl<float>::MAX_SIN_PHI, o2::base::PropagatorImpl<float>::MAX_STEP, mCorrType)) {
+    if (!propInstance->propagateTo(track, trackingHit.xTrackingFrame, useBxByBz, o2::base::PropagatorImpl<float>::MAX_SIN_PHI, o2::base::PropagatorImpl<float>::MAX_STEP, mCorrType)) {
```

Comment:
Actually, should be `!useBxByBz` since `true` in this method means use BZ only.

Reply:
right, test is now running, tomorrow I'll remove the WIP